RoryMercury.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
filename should be in all lower case https://github.com/yourWaifu/is-your-waifu-legal/tree/master/waifus